### DISCUSSION TITLE
Shellbar Search Placement

### DISCUSSION THREAD

**Discussion by kagos** [UI5/webcomponents#6593] [Q&A]
Per the Fiori Horizon specifications outlined in the Visual Core wiki, the search field is centered for XL screens. Does the UI5 WC Shellbar component support the search position change?

https://wiki.one.int.sap/wiki/pages/viewpage.action?pageId=2697898394

**Comment by MarcusNotheis** [comment]
Hey @kagos 
thanks for reaching out! As the `ShellBar` is developed in the UI5 Web Components repository, I'll transfer this discussion to this repo.
